ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -62,7 +62,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-disc-priv-tp-f28
```

Comment:
I think we still need to keep the stable z-stream on f28, there are not many jobs on stable z-stream upgrade ci.

Reply:
let me keep `gcp-ipi-proxy-tp` at `f28` and leave others at `f60` because all aws jobs are disconnected and `nutanix` have resource limitation issue.

Reply:
I suggest also keep `aws-ipi-disc-priv-tp` at f28, because there is OTA case enabled on it. f60 sounds like too long.